content/user/advanced-modules/PDFTemplates.adoc

```diff
@@ -0,0 +1,116 @@
+---
+Title: PDF-Templates
+Weight: 85
+---
+
+
+:toc:
+:toc-title: Table of content
```

Comment:
You can leave out this `toc-title` property, it is only useful in the Russian version. In English, the default "Table of contents" will be used.